### DISCUSSION TITLE
No dependencies found in `settings.gradle.kts`

### DISCUSSION THREAD

**Discussion by brizzbuzz** [dipien/releases-hub-gradle-plugin#128] [General]
**Describe the bug**

Hi... I would really like to use this plugin for several applications I'm working on.  However, it seems to not be picking up _any_ dependencies from a `settings.gradle.kts`.  Not sure why as it is explicitly listed as supported, and I have plenty of things in my version catalogs.  

Brief snippet from my `settings.gradle.kts`

```kotlin
// ...
dependencyResolutionManagement {
    versionCatalogs {
        create("plugin") {
            alias("artifactory").toPluginId("com.jfrog.artifactory").version(artifactoryVersion)
            alias("coveralls").toPluginId("com.github.kt3k.coveralls").version(coverallsPluginVersion)
            alias("detekt").toPluginId("io.gitlab.arturbosch.detekt").version(detektVersion)
            alias("flyway").toPluginId("org.flywaydb.flyway").version(flywayVersion)
            alias("git-hooks").toPluginId("com.github.jakemarsden.git-hooks").version(gitHooksPluginVersion)
            alias("kotlin").toPluginId("org.jetbrains.kotlin.jvm").version(kotlinVersion)
            alias("processes").toPluginId("com.github.johnrengelman.processes").version(processesPluginVersion)
            alias("serialization").toPluginId("org.jetbrains.kotlin.plugin.serialization").version(kotlinVersion)
            alias("test-logger").toPluginId("com.adarshr.test-logger").version(testLoggerVersion)
            alias("release-hub").toPluginId("com.dipien.releaseshub.gradle.plugin").version(releaseHubVersion)
            alias("spring-boot").toPluginId("org.springframework.boot").version(springBootVersion)
            alias("spring-kotlin").toPluginId("org.jetbrains.kotlin.plugin.spring").version(kotlinVersion)
            alias("spring-dependency-manager").toPluginId("io.spring.dependency-management")
                .version(springDependencyManagerVersion)
            alias("spring-doc").toPluginId("org.springdoc.openapi-gradle-plugin").version(springDocPluginVersion)
        }
        create("kt") {
            alias("coroutines-core").to(Groups.kotlinx, "kotlinx-coroutines-core").version(kotlinCoroutineVersion)
            alias("coroutines-reactor").to(Groups.kotlinx, "kotlinx-coroutines-reactor").version(kotlinCoroutineVersion)
            alias("serialization").to(Groups.kotlinx, "kotlinx-serialization-json").version(kotlinxSerializationVersion)

            bundle("coroutines", listOf("coroutines-core", "coroutines-reactor"))
        }
        // More catalogs ...
    }
}
```

However, when i run `./gradlew listDependencies` I see the following 

```
settings.gradle.kts
 - No dependencies found here

gradle/wrapper/gradle-wrapper.properties
 - gradle:7.3

build.gradle.kts
 - No dependencies found here

api/build.gradle.kts
 - ${it.module.group}:${it.module.name}:${it.versionConstraint}

domain/build.gradle.kts
 - No dependencies found here
```

**To Reproduce**
have version catalog in gradle settings file 

**Expected behavior**
This should list all dependencies that exist in the settings version catalogs 

**Screenshots**

**Additional context**

**Comment by maxirosson** [comment]
Hi. The plugin only supports the basic notation: “groupId:artifactId:version”. The [versions] tag on the libs.versions.toml file or the versionRef on the settings.gradle.kts are not supported yet. It will be implemented as part of this [issue](https://github.com/dipien/releases-hub-gradle-plugin/issues/17).

This missing feature is not urgent in my opinion, because given that you are automating all the dependencies upgrades, you don't need to define variables for your versions.

**Reply by brizzbuzz** [reply]
I'm confused... I'm not using `versionRef`. I'm just using `version()`, although my versions are delegated to the `gradle.properties` file.  not sure if that is the problem here?

**Comment by maxirosson** [comment]
You need to use this format:

`alias("firebase-core").to("com.google.firebase:firebase-core:1.0.0")`

**Reply by brizzbuzz** [reply]
Ahhh... well.  I can't go across 50+ code bases and change all this, so I'll probably have to wait for this feature to be implemented. I'll try to find some time to pull this codebase and see if I can get an implementation working for my use case :) 

Really appreciate the speedy responses!
